app/util/job.ts

```diff
@@ -22,6 +22,35 @@ async function cleanupWorkItemsForJobID(jobID: string, logger: Logger): Promise<
   }
 }
 
+/**
+   * Set the state of a job to 'paused' unless already in a terminal state then save it.
+   * 
+   * @param tx - the transaction to perform the updates with
+   * @param job - the job to save and update
+   * @param logger - the logger to use for logging errors/info
+   * @param message - the job's paused message
+   * @throws {@link ConflictError} if the job is already in a terminal state.
+ */
+export async function pauseAndSaveJob(
+  tx: Transaction,
+  job: Job,
+  finalStatus: JobStatus,
+  logger: Logger,
+  message?,
+): Promise<void> {
+  try {
+    if (job.status != JobStatus.RUNNING) {
```

Comment:
`pause` which is called on line 45 already does this validation - can remove it from here.

Reply:
done